plugins/check_swap.c

```diff
@@ -518,8 +518,14 @@ process_arguments (int argc, char **argv)
 int
 validate_arguments (void)
 {
-	if (have_crit == FALSE && have_warn == FALSE)
-		return ERROR;
+	if (have_warn > 0 && have_crit == 0) {
+		usage4
+			(_("Must define both warning and critical thresholds"));
+	}
+	else if (have_crit > 0 && have_warn == 0) {
+		usage4
+			(_("Must define both warning and critical thresholds"));
+	}
```

Comment:
I haven't tested this code yet, but are we changing the plugin so that you're required to specify both a warning and critical threshold? Or did the plugin not work / crash when only one threshold was specified?

Reply:
The code works:

```
 ./check_swap -w 90
check_swap: Must define both warning and critical thresholds
Usage:
 check_swap [-av] [-w <percent_free>%] [-c <percent_free>%]
  [-w <bytes_free>] [-c <bytes_free>] [-n <state>]

$ ./check_swap -c 90
check_swap: Must define both warning and critical thresholds
Usage:
 check_swap [-av] [-w <percent_free>%] [-c <percent_free>%]
  [-w <bytes_free>] [-c <bytes_free>] [-n <state>]

$ ./check_swap -w 95 -c 90
SWAP OK - 100% free (4091 MB out of 4091 MB) |swap=4091MB;0;0;0;4091
```

Prior to this PR the check_swap plugin did not work when
- only one threshold was specified
- no thresholds were specified